centrallix/include/expression.h

```diff
@@ -307,6 +307,7 @@ int exp_internal_SetupControl(pExpression exp);
 pExpControl exp_internal_LinkControl(pExpControl ctl);
 int exp_internal_UnlinkControl(pExpControl ctl);
 
+void meta_double_metaphone(const char* str, char** primary_code, char** secondary_code);
```

Comment:
Recommend moving the double metaphone library code to a file in the utility directory in centrallix.  Otherwise the function here with a different prefix than the exp module looks odd.